application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -331,7 +334,33 @@ public function checkRequiredStages(Validation $validation, $completed_stages, $
 	 * @param  Array      $attributes
 	 * @param  Array      $fullData
 	 */
-	public function checkRequiredAttributes(Validation $validation, $attributes, $fullData)
+	public function checkRequiredPostAttributes(Validation $validation, $attributes, $fullData)
+	{
```

Comment:
Hrm... I don't even know what should happen with them. Technically required attributes aren't required until their stage is 'complete' ... but thats completely counter intuitive.

I'll defer solving that issue to later. This is still much better than before.